friends/src/components/Item.js

```diff
@@ -0,0 +1,103 @@
+import React, { Component, Fragment } from 'react'
+import axios from 'axios'
+import { Line } from '../styles/Card'
+import { Input } from '../styles/Item'
+
+class Item extends Component {
+  state = {
+    edit: false,
+    value: this.props.children
+  }
+
+  // create ref to set focus on input when it loads
+  textInput = React.createRef()
+
+  // add listener to check for clicks outside element
+  componentDidMount() {
+    document.addEventListener('mousedown', this.handleClickOutside)
+  }
+
+  // set focus on input if editing
+  componentDidUpdate() {
+    if (this.state.edit) {
+      this.focusTextInput()
+    }
+  }
+
+  // remove listenr
+  componentWillUnmount() {
+    document.removeEventListener('mousedown', this.handleClickOutside)
+  }
+
+  // set wrapper ref for listener
+  setWrapperRef = node => {
+    this.wrapperRef = node
+  }
+
+  // update state if click registers outside element
+  handleClickOutside = event => {
+    if (this.wrapperRef && !this.wrapperRef.contains(event.target)) {
+      this.setState({
+        edit: false,
+        value: this.props.children
+      })
+    }
+  }
+
+  // set focus on input with ref
+  focusTextInput() {
+    this.textInput.current.focus()
+  }
+
+  // load input field on click
+  handleClick = event => {
+    this.setState({ edit: true })
+  }
+
+  // update state in response to input field changes
+  handleChange = event => {
+    this.setState({ value: event.target.value })
+  }
+
+  // update field in db on enter (submit)
+  handleSubmit = event => {
+    event.preventDefault()
+    const { id, field, updateFriends } = this.props
+
+    axios
+      .put(`/friends/${id}`, { [field]: this.state.value })
+      .then(res => {
+        updateFriends(res.data)
+        // close input and remove focus
+        this.resetElement()
+      })
+      .catch(err => console.log(err))
+  }
+
+  resetElement = () => {
+    this.setState({
+      edit: false,
+      value: this.props.children
+    })
+  }
+
+  render() {
+    const { children } = this.props
+    const { edit, value } = this.state
+    const { handleClick, handleChange, handleSubmit } = this
+
+    return (
+      <Fragment>
```

Comment:
Same thing, just depends on whether you've imported it directly or not. `import React, { Fragment } from 'react'` allows you to use `<Fragment>`, otherwise you have to use `<React.Fragment>`